service-front/module/Application/src/Helpers/LpaFormHelper.php

```diff
@@ -0,0 +1,263 @@
+            $channelCheck = $this->checkChannel($siriusCheck);
+
+            if ($statusCheck['error'] === true) {
+                $result['status'] = $statusCheck['status'];
+                $result['message'] = $statusCheck['message'];
+            } elseif ($idCheck['error'] === true) {
+                $result['status'] = 'no match';
+                $result['message'] = $idCheck['message'];
+                $result['additional_data'] = [
+                    'name' => $idCheck['name'],
+                    'address' => $idCheck['address'],
+                    'name_match' => $idCheck['name_match'],
+                    'address_match' => $idCheck['address_match'],
+                    'error' => $idCheck['error']
+                ];
+                $result['data'] = [
+                    "case_uuid" => $uuid,
+                    "LPA_Number" => $form->get('lpa')->getValue(),
+                    "Type_Of_LPA" => $this->getLpaTypeFromSiriusResponse($siriusCheck),
+                    "Donor" => $this->getDonorNameFromSiriusResponse($siriusCheck),
+                    "Status" => $statusCheck['status'],
+                    "CP_Name" => $detailsData['firstName'] . " " . $detailsData['lastName'],
+                    "CP_Address" => $detailsData['address']
+                ];
+            } elseif ($channelCheck['error'] === true) {
+                $result['status'] = $channelCheck['status'];
+                $result['message'] = $channelCheck['message'];
+                $result['data'] = [
+                    "case_uuid" => $uuid,
+                    "LPA_Number" => $form->get('lpa')->getValue(),
+                    "Type_Of_LPA" => $this->getLpaTypeFromSiriusResponse($siriusCheck),
+                    "Donor" => $this->getDonorNameFromSiriusResponse($siriusCheck),
+                    "Status" => $statusCheck['status'],
+                    "CP_Name" => $detailsData['firstName'] . " " . $detailsData['lastName'],
+                    "CP_Address" => $detailsData['address']
+                ];
+            } else {
+                $result['status'] = "Success";
+                $result['message'] = "";
+                $result['data'] = [
+                    "case_uuid" => $uuid,
+                    "LPA_Number" => $form->get('lpa')->getValue(),
+                    "Type_Of_LPA" => $this->getLpaTypeFromSiriusResponse($siriusCheck),
+                    "Donor" => $this->getDonorNameFromSiriusResponse($siriusCheck),
+                    "Status" => $statusCheck['status'],
+                    "CP_Name" => $detailsData['firstName'] . " " . $detailsData['lastName'],
+                    "CP_Address" => $detailsData['address']
+                ];
+            }
+        }
+
+        return new LpaFormHelperResponseDto(
+            $uuid,
+            $form,
+            [
+                'lpa_response' => $result
+            ],
+        );
+    }
+
+    public function getCpAddressFromSiriusResponse(array $siriusCheck): array
```

Comment:
It looks like this and every other method below should be private, since `findLpa` is the only one used by anything external.
```suggestion
    private function getCpAddressFromSiriusResponse(array $siriusCheck): array
```

Reply:
The problem is that the unit tests then fail if I make them private